### ISSUE TITLE
Cannot open or close indexes on HTTPS ES endpoint

### ISSUE THREAD

**Issue by anhlqn** [lmenezes/cerebro#287]
**Elasticsearch version:** 5.6.8
**cerebro:** 0.7.2

I use IIS on Windows 2012 R2 as a proxy in front of ES to allow only HEAD, POST, and GET verbs to prevent template changes and index deletion. The IIS site for ES is configured with HTTPS, and cerebro can connect to ES after I follow the guide to trust the Root CA certificate in #6 . However, after connected, I cannot open or close indexes. The error message is `Operation failed` and the following details are in Chrome Dev Tools
```
body:{,…}
error:
"<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN""http://www.w3.org/TR/html4/strict.dtd">
<HTML><HEAD><TITLE>Length Required</TITLE>
<META HTTP-EQUIV="Content-Type" Content="text/html; charset=us-ascii"></HEAD>
<BODY><h2>Length Required</h2>
<hr><p>HTTP Error 411. The request must be chunked or have a content length.</p>
</BODY></HTML>
"
status: 411
```

cerebro works fine if I connect directly to the ES RESTAPI. Could anyone help me figure out this issue?

Thanks,

**Comment by lmenezes** [comment]
@anhlqn Is this still an issue? And does this happens only with close/open indices, and the rest of actions work fine?

**Comment by anhlqn** [comment]
@lmenezes Yes, I just checked. GET and PUT work fine (I can toggle allocation and update index templates if I allow PUT in IIS), but POST doesn't work.

**Comment by lmenezes** [comment]
@anhlqn and you have the same problem if you try a search request(using POST), through the rest client?

**Comment by anhlqn** [comment]
I'm able to search as well as open and close indexes via the rest client interface.
